src/modules/Dropdown/Dropdown.js

```diff
@@ -747,7 +802,13 @@ export default class Dropdown extends Component {
         active={isActive(opt.value)}
         onClick={this.handleItemClick}
         selected={selectedIndex === i}
-        text={opt.text}
+        text={
+          allowAdditions
+          && (additionPosition === 'top' ? i === 0 : i === options.length - 1)
+          && opt.text.startsWith('Add: ') ?
+          <span><strong><em>Add:</em></strong> {opt.text.substring(5)}</span> :
+          opt.text
+        }
```

Comment:
This was also true before, but the extra thing I do here is add some styling to the `Add:` part, such that it stands out from the rest.